api/src/main/java/org/openmrs/Cohort.java

```diff
@@ -273,15 +326,23 @@ public String getName() {
 	public void setName(String name) {
 		this.name = name;
 	}
-	
+
 	public Set<Integer> getMemberIds() {
 		return memberIds;
 	}
-	
+
 	public void setMemberIds(Set<Integer> memberIds) {
 		this.memberIds = new TreeSet<Integer>(memberIds);
 	}
-	
+
+	public Collection<CohortMembership> getMembers() {
+		return members;
```

Comment:
Our convention is such that collection getters never return null, you need to check if members is null and if yes you instantiate it